### PR TITLE
Bump kubernetes

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
/cc @aveshagarwal 

Bumped to kube-1.14. Passing unit-tests locally. Let's see if it passes e2es..

### CODE REVIEW

**Comment by ravisantoshgudimetla** [comment]
ping @aveshagarwal..

File: pkg/descheduler/pod/pods_test.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ func TestPodTypes(t *testing.T) {
 	p4 := test.BuildTestPod("p4", 400, 0, n1.Name)
 	p5 := test.BuildTestPod("p5", 400, 0, n1.Name)
 	p6 := test.BuildTestPod("p6", 400, 0, n1.Name)
-	p6.Spec.Containers[0].Resources.Requests[v1.ResourceNvidiaGPU] = *resource.NewMilliQuantity(3, resource.DecimalSI)
+	p6.Spec.Containers[0].Resources.Limits[NVIDIAGPUResourceName] = *resource.NewMilliQuantity(3, resource.DecimalSI)
```
Comment (aveshagarwal) [inline pkg/descheduler/pod/pods_test.go]: curious why it changed from requests to limits?

File: pkg/descheduler/evictions/evictions.go [hunk 2]
```diff
@@ -45,7 +45,7 @@ func EvictPod(client clientset.Interface, pod *v1.Pod, policyGroupVersion string
 		},
 		DeleteOptions: deleteOptions,
 	}
-	err := client.Policy().Evictions(eviction.Namespace).Evict(eviction)
+	err := client.PolicyV1beta1().Evictions(eviction.Namespace).Evict(eviction)
```
Comment (aveshagarwal) [inline pkg/descheduler/evictions/evictions.go]: from policy to policyv1beta1?

**Comment by aveshagarwal** [comment]
just left minor comments.
yes please rebase.

**Pull request closed without merging**